components/scream/src/physics/p3/micro_p3_iso_c.f90

```diff
@@ -9,6 +9,50 @@ module micro_p3_iso_c
 # define c_real c_float
 #endif
 
+!
+! This file contains bridges between micro_p3 fortran and scream c++ p3. subroutines ending in
+! "_c" are intended to used as a c->fortran bridge, subroutines ending in "_f_" are intended to be
+! used a fortran->c bridge.
+!
+
+interface
+
+  subroutine find_lookuptable_indices_1a_f(dumi,dumjj,dumii,dumzz,dum1,dum4,dum5,dum6,      &
+       qitot,nitot,qirim,rhop) bind(C)
+    use iso_c_binding
+
+    ! arguments:
+    integer(kind=c_int), intent(out) :: dumi,dumjj,dumii,dumzz
+    real(kind=c_real),   intent(out) :: dum1,dum4,dum5,dum6
+    real(kind=c_real),   value, intent(in)  :: qitot,nitot,qirim,rhop
+  end subroutine find_lookuptable_indices_1a_f
+
+  subroutine find_lookuptable_indices_1b_f(dumj,dum3,qr,nr) bind(C)
+    use iso_c_binding
+
+    integer(kind=c_int), intent(out) :: dumj
+    real(kind=c_real),   intent(out) :: dum3
+    real(kind=c_real),   intent(in) :: qr, nr
+  end subroutine find_lookuptable_indices_1b_f
+
+  subroutine access_lookup_table_f(dumjj,dumii,dumi,index,dum1,dum4,dum5,proc) bind(C)
+    use iso_c_binding
+
+    integer(kind=c_int), intent(in) :: dumjj, dumii, dumi, index
+    real(kind=c_real),   intent(in) :: dum1, dum4, dum5
+    real(kind=c_real),   intent(out) :: proc
+  end subroutine access_lookup_table_f
+
+  subroutine access_lookup_table_coll_f(dumjj,dumii,dumj,dumi,index,dum1,dum3,dum4,dum5,proc) bind(C)
+    use iso_c_binding
+
+    integer(kind=c_int), intent(in) :: dumjj,dumii,dumj,dumi,index
+    real(kind=c_real),   intent(in) :: dum1,dum3,dum4,dum5
+    real(kind=c_real),   intent(out) :: proc
+  end subroutine access_lookup_table_coll_f
+
+end interface
```

Comment:
@jgfouca - I don't understand why these "_f" subroutines are in the "iso_c" file. They seem to be redundant with identical subroutines in the "iso_f" file, which seems like a more reasonable place for them...

Reply:
Thanks @PeterCaldwell , I clearly forgot to remove from duplicated code from this file.